src/diffusers/models/unet_2d_blocks.py

```diff
@@ -2051,10 +2065,15 @@ def __init__(
         else:
             self.upsamplers = None
 
-    def forward(self, hidden_states):
+    def forward(self, hidden_states, zq):
         for resnet, attn in zip(self.resnets, self.attentions):
-            hidden_states = resnet(hidden_states, temb=None)
-            hidden_states = attn(hidden_states)
+            if self.use_spatial_norm:
```

Comment:
again, is it this condition necessary? can we just make sure we aways pass  `zq` as  `None` to a decoder block without spatial norm?